evalml/tests/automl_tests/test_automl.py

```diff
@@ -645,6 +645,32 @@ def generate_fake_dataset(rows):
     assert automl.data_split.test_size == (automl._LARGE_DATA_PERCENT_VALIDATION)
 
 
+def test_data_split_shuffle():
+    # this test checks that the default data split strategy should shuffle data. it creates a target which
+    # increases monotonically from 0 to n-1.
+    #
+    # if shuffle is enabled, the baseline model, which predicts the mean of the training data, should accurately
+    # predict the mean of the validation data, because the training split in each CV fold will contain a mix of
+    # values from across the target range, thus yielding an R^2 of close to 0.
+    #
+    # if shuffle is disabled, the mean value learned on each CV fold's training data will be incredible inaccurate,
+    # thus yielding an R^2 well below 0.
+
+    n = 100000
```

Comment:
If this test doesn't take too long it might be worth running it with n=1000 so that the KFold CV is used?

Reply:
Ah shoot you're right. I'll track this down. I upped the number because I wasn't reproing it as reliably at lower values.